src/components/styles/StyledLink.tsx

```diff
@@ -1,32 +1,22 @@
 import { css } from "@emotion/react";
-import styled from "@emotion/styled";
-import { palette } from "@leafygreen-ui/palette";
+import { Link as LGLink } from "@leafygreen-ui/typography";
 import { Link } from "react-router-dom";
-import { fontSize } from "constants/tokens";
 
-const { blue } = palette;
+export const StyledLink = (props) => (
+  <LGLink
```

Comment:
Minor nit but Icon links have the underline effect 
<img width="344" alt="image" src="https://user-images.githubusercontent.com/4605522/198695284-94f56a50-d9e5-4804-8bbd-425c70ae67a7.png">